corehq/apps/locations/tests/test_views.py

```diff
@@ -152,3 +155,49 @@ def test_invalid_remove_has_users(self, _):
         data = {'loc_types': [loc_type1, loc_type2]}
         with self.assertRaises(LocationConsistencyError):
             self.send_request(data)
+
+
+class LocationSearchViewTest(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super(LocationSearchViewTest, cls).setUpClass()
+        delete_all_users()
+        cls.domain = "test-domain"
+        cls.project = create_domain(cls.domain)
+        cls.username = "request-er"
+        cls.password = "foobar"
+        cls.web_user = WebUser.create(cls.domain, cls.username, cls.password, None, None)
+        cls.web_user.add_domain_membership(cls.domain, is_admin=True)
+        cls.web_user.set_role(cls.domain, "admin")
+        cls.web_user.save()
+        cls.loc_type1 = LocationType(domain=cls.domain, name='type1', code='code1')
+        cls.loc_type1.save()
+        cls.loc1 = make_loc(
+            'loc_1', type=cls.loc_type1, domain=cls.domain
+        )
+        cls.loc2 = make_loc(
+            'loc_2', type=cls.loc_type1, domain=cls.domain
+        )
+
+    def setUp(self):
+        self.client.login(username=self.username, password=self.password)
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.project.delete()
+        delete_all_users()
+        return super().tearDownClass()
+
+    @mock.patch('django_prbac.decorators.has_privilege', return_value=True)
```

Comment:
FWIW there's also a `@privilege_enabled` decorator you can use for this to target a specific privilege